index.css

```diff
@@ -3,4 +3,20 @@ display: flex;
 align-items: center;
 justify-content: center;
 background-color: gray;
+}
+
+.main-button {
+color: #FFFFFF;
+text-align: center;
+background-color: #111B47;
+width: 190px;
+height: 36px;
+}
+
+.second-button {
+background-color: rgba(255, 255, 255, 0);
+color: #091133;
+text-align: center;
+width: 190px;

```

Comment:
when we have long text it will break design. change width into padding-left, padding-right